ReactCommon/cxxreact/JSExecutor.h

```diff
@@ -53,10 +53,14 @@ class JSExecutorFactory {
 
 class RN_EXPORT JSExecutor {
 public:
+  /**
+   * Sets all globals in the JS context.
```

Comment:
Maybe something like: "Prepares the JS runtime for React Native by installing global variables. Called once before any JS is evaluated."